lib/public/views/Tags/Tags.js

```diff
@@ -75,6 +75,24 @@ class TagModel extends Observable {
         return this.text;
     }
 
+    /**
+     * Getter for email
+     *
+     * @returns {String} The text currently inserted in the tag creation screen.
+     */
+    getEmail() {
+        return this.email;
```

Comment:
Similar to the RUNS and LOGS refactoring, I would suggest we start using the standard convention for JS getters and setters